tests/integration/mixer/envoymixer/testdata/envoyconfig.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Can you clean up this file? Including rename it to something meaningful and remove ext authz part because it is not really tested.